### PR TITLE
Dbudzinski/update dev docs

### PR DESCRIPTION
**Pull request by budzworthy**
This commit updates the developer documentation to be more in line with the JW player public repo

## Description

The goal of these doc updates are to more thoroughly describe the actual latest state of the repo as well as to better position the repo for receiving public contributions in the way that we want. I copied a lot of the readme, code of conduct, and contributing guide from the JW player repo, because it makes sense to have similar processes for all of the JW public facing repos.

This PR solves [BC-118](https://jwplayer.atlassian.net/browse/BC-118)

### Steps completed:

According to our definition of done, I have completed the following steps:

- [x] User stories met
- [x] Storybook stories
- [ ] Unit tests added
- [ ] Linting passing
- [ ] Unit tests passing
- [x] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [x] Rebased & ready to merge without conflicts
- [x] Reviewed own code

### CODE REVIEW

File: docs/backend-services.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Backend dependencies and architecture
+
+The application is built as a single page web app that can run without its own dedicated backend. This is useful for
+hosting it with a very simple, static host, like github pages. The server serves the static web content and the frontend
+calls the [JW player platform Delivery API](https://developer.jwplayer.com/jwplayer/docs) directly.
```
Comment (emptyflash) [inline docs/backend-services.md]: Small nitpick
```suggestion
calls the [JW Player Platform Delivery API](https://developer.jwplayer.com/jwplayer/docs) directly.
```

**Review by emptyflash — APPROVED** [review]

File: docs/backend-services.md (hunk 1 above)
Comment (kcorneli201) [reply docs/backend-services.md]:
```suggestion
calls the [JW Player Delivery API](https://developer.jwplayer.com/jwplayer/docs) directly.
```

Comment (kcorneli201) [reply docs/backend-services.md]: @dbudzins @emptyflash The Management API is a the  _Platform_ Management API. The other API is merely Delivery API.

**Review by kcorneli201 — CHANGES_REQUESTED** [review]
@dbudzins I added one minor update.

**Pull request merged**